.circleci/config.yml

```
@@ -93,7 +93,7 @@ jobs:
       - run:
           name: Install Brain extraction tests
           command: |
-            datalad install -r https://github.com/nipreps-data/brain-extraction-tests.git
+            datalad install -r https://gin.g-node.org/nipreps-data/brain-extraction-tests.git
```

For whatever reason, when datalad (or maybe git-annex) sees `.git` on GIN, it will disable the annex.

```suggestion
            datalad install -r https://gin.g-node.org/nipreps-data/brain-extraction-tests
```